docs/woofhud.md

```diff
@@ -0,0 +1,84 @@
+# WOOFHUD
+
+Woof (since version 10.6) supports the WOOFHUD lump to customize the appearance of the extended Boom HUD.
+
+## Description
+
+The Boom HUD shows information about the player's health, armor, weapons, ammo and keys using different widgets, i.e. lines of text and symbols. It is usually made visible by hitting the <kbd>F5</kbd> key and repeatedly hitting the <kbd>F5</kbd> key toggles through three different modes: the "minimal" mode which does not show any information by default, the "compact" mode which shows all information in the lower left corner of the screen and the "distributed" mode which shows information spread across the corners of the screen.
+
+The WOOFHUD lump can be used to modify the positions of these widgets for each mode.
+
+## Format
+
+The WOOFHUD lump is in text format and consists of paragraphs, which are separated by blank lines. Each paragraph begins with a line starting with the keyword `hud` and a number for the HUD mode which is to be modified: `0` for minimal, `1` for compact and `2` for distributed.
+The following lines start with the name of the HUD widget which is to be positioned followed by either a keyword giving the position relative to the screen or two numbers giving the absolute X and Y screen coordinates.
+
+Possible values for the HUD widget names:
+
+ * "title" or "levelname"
+ * "armor"
+ * "health"
+ * "ammo"
+ * "weapon" or "weapons"
+ * "keys"
+ * "monsec" or "stats"
+ * "sttime" or "time"
+ * "coord" or "coords"
+ * "fps" or "rate"
+
+Possible values for the widget position keywords:
+
+ * "topleft" or "upperleft"
+ * "topright" or "upperright"
+ * "bottomleft" or "lowerleft"
+ * "bottomright" or "lowerright"
+
+When using relative screen positioning, the widgets are aligned "first come, first serve". For example, the first widget in a paragraph that is aligned with the "bottomleft" keyword will end up in the very bottom-left area of the screen and each following widget that is aligned with the same keyword will get stacked one line above.
+
+Absolute X and Y screen coordinates are limited to the low-resolution non-widescreen visible area of the screen, i.e. 0 <= X < 320 and 0 <= Y < 200. Negative values get interpreted relative to the right or lower edges of the screen, respectively.
+
+## Example
```

Comment:
Maybe add a minimal example that uses the alternate absolute coordinate arguments.